src/analytics/private/sqls/redshift/sp-clear-item-and-user.sql

```diff
@@ -0,0 +1,37 @@
+CREATE OR REPLACE PROCEDURE {{schema}}.sp_clear_item_and_user() 
```

Comment:
What's the logic to clear the legacy items and users?

Reply:
remove older user/item for the same id, keep the latest one based on `event_timestamp`